.github/license-lint-config.yml

```diff
@@ -41,4 +41,6 @@ allowlisted_modules:
   # Apache 2.0: https://github.com/apparentlymart/go-textseg/blob/5b41aa275ccaac4ccaa91729a8ee92e7eac9c728/LICENSE
   - github.com/apparentlymart/go-textseg/v13
   # Apache 2.0: https://github.com/apparentlymart/go-textseg/tree/72b78f42484ddc3f58858f794da1771fb9559ad0/LICENSE
-  - github.com/apparentlymart/go-textseg/v15
\ No newline at end of file
+  - github.com/apparentlymart/go-textseg/v15
+  # todo
+  - github.com/gke-labs/multicluster-leader-election
```

Comment:
will need to specify the license in https://github.com/gke-labs/multicluster-leader-election and follow up here

Reply:
I approved so we should be able to rebase